src/js/media-controller.js

```diff
@@ -238,6 +238,10 @@ class MediaController extends MediaContainer {
       },
       'ratechange': () => {
         this.propagateMediaState(MediaUIAttributes.MEDIA_PLAYBACK_RATE, this.media.playbackRate);
+      },
+      'waiting,stalled,playing': () => {
+        const isLoading = !this.media?.paused && this.media?.readyState < 3;
```

Comment:
proposal: Let's not conflate `loading` state with `paused` state. Instead, let's have `<media-loading-indicator/>` accept both `MediaUIAttributes .MEDIA_PAUSED` and `MediaUIAttributes.MEDIA_LOADING` (which it already does) and by default style it to show only when `loading` and not `paused`.